server/src/main/java/org/spine3/server/aggregate/AggregateRoot.java

```diff
@@ -79,11 +79,15 @@ protected BoundedContext getBoundedContext() {
      * Obtains a part state by its class.
      *
      * @param partStateClass the class of the state of the part
-     * @param <S> the type of the part state
+     * @param <S>            the type of the part state
      * @return the state of the part or a default state if the state was not found
+     * @throws IllegalStateException if a repository was not found,
+     *                               or the repository does not match expectations
```

Comment:
> expectations.

Please let readers know what they are.

That's a common issue, please address the similar cases.